ldap/ldif/template-dse.ldif.in

```diff
@@ -50,6 +50,14 @@ objectclass: top
 objectclass: nsContainer
 cn: features
 
+dn: oid=1.2.840.113556.1.4.1941,cn=features,cn=config
+objectClass: top
+objectClass: directoryServerFeature
+oid: 1.2.840.113556.1.4.1941
+cn: InChain Matching Rule
+aci: (targetattr != "aci")(version 3.0; acl "InChain Matching Rule"; allow( rea
```

Comment:
I don't think everyone should be able to trigger this. This is a potential DOS vector, and so use of this rule should be extremely limited to trusted users.

Reply:
I had similar concern. Checking that permission on that entry let the administrator able to restrict use of InChain MR to specific users/groups/roles...
The default value is anyone being logged. Do you think it should be more restrictive by default ?

Reply:
Isn't the current default "ldap://anyone"?  Like you said, I think it should be at least "ldap://all"

Reply:
> Isn't the current default "ldap://anyone"? Like you said, I think it should be at least "ldap://all"

Nevermind, either you just changed it, or I thought it "anyone" by mistake.

Reply:
Since this feature is only intended for the global catalog, IMHO the safest solution is to have a "deny all" rule by default (so only directory manager will be able to use it). If freeipa needs something else they can easily overwrite the aci in their setup phase

Reply:
Sorry for the confusion. I was thinking allowing InChain MR to any authenticated users. (reject anonymous access).
Regarding restricting the use of the feature by default. From a performance pov it is a valid point but in the opposite it already exists expensive control, like deref, that is freely supported.
@Firstyear were you thinking of a DOS by expensive requests attack ?

Reply:
Yes - This control is explicitly advised *against* by MS on AD, and by AD admins. We should limit it as much as possible.

Just because we allowed an expensive control in the past, doesn't mean we should allow new expensive controls to be a free-for-all.

Reply:
Yes that make sense. Sorry for this long thread I am now convinced to restrict this MR. I will go with @progier389 approach, changing the aci with a deny right.

Reply:
Thank you

Reply:
@Firstyear , I changed the right of the aci that is now DENY for anyone. Do you mind having a new review ?